api/src/main/java/io/strimzi/api/kafka/model/common/template/Volume.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.common.template;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.CSIVolumeSource;
+import io.fabric8.kubernetes.api.model.ConfigMapVolumeSource;
+import io.fabric8.kubernetes.api.model.EmptyDirVolumeSource;
+//import io.fabric8.kubernetes.api.model.EmptyDirVolumeSource;
+import io.fabric8.kubernetes.api.model.SecretVolumeSource;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.KubeLink;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of additional volumes for Strimzi resources.
+ */
+@Buildable(editableEnabled = false, builderPackage = Constants.FABRIC8_KUBERNETES_API)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "name", "path", "subPath", "secret", "configMap", "emptyDir", "csi" })
+@EqualsAndHashCode
+@ToString
+public class Volume implements UnknownPropertyPreserving {
```

Comment:
Are we sure about this naming. It could be confused with the native kubernetes volume. 

Suggestions

- AdditionalVolume
- KafkaVolume

Reply:
Yes, I agree, changed to AdditionalVolume and using @JsonProperty annotation to use "volumes" in the CRD (as per comment in proposal review) while keeping additionalVolumes in the code